lib/app_web/controllers/page_controller.ex

```diff
@@ -18,4 +18,10 @@ defmodule AppWeb.PageController do
   def ping(conn, params) do
     Ping.render_pixel(conn, params)
   end
+
+  def logout(conn, _params) do
+    conn
+    |> AuthPlug.logout()
```

Comment:
This is the interesting part. 💡 
With this single invocation of `AuthPlug.logout/1` all "logout" action is handled transparently.

Reply:
I was wondering if the logout function could failed before rendering the `loggedout.html` page.
I've created an issue on auth_plug: https://github.com/dwyl/auth_plug/issues/38

Reply:
Excellent idea to check for errors. Thanks.